resources/nginx.conf

```
@@ -31,7 +31,7 @@ server {
 server {
   listen       80;
   listen  [::]:80;
-  server_name  keymanweb.com.localhost;
+  server_name  ~^(web.)?keyman(web)?.com.localhost;
```

can we instead do `~^(web.keyman.com.localhost|keymanweb.com.localhost);` -- easier to read and doesn't allow web.keymanweb.com.localhost which is redundant and actually doesn't map on prod? Also do we need to escape `.`?

```suggestion
  server_name  ~^(web\.keyman\.com|keymanweb\.com)\.localhost;
```

---

How about `~^(web\.keyman|keymanweb).com.localhost;`

---

ok